installer/tue-install.bash

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+_tue-check-env-vars || return 1
+
+# Update installer
+if [ ! -d $TUE_DIR ]
+then
+    git clone https://github.com/tue-robotics/tue-env.git $TUE_DIR
+elif [[ -n "$CI" ]] #Do not update with continuous integration but do fetch to refresh available branches
+then
+    echo -en "Fetching tue-get... "
+    git -C $TUE_DIR fetch
+else
+    echo -en "Updating tue-get... "
+    git -C $TUE_DIR pull --ff-only --prune
+
+    error_code=$?
+
+    if [ ! $error_code -eq 0 ]
+    then
+        # prompt for conformation
+        exec < /dev/tty
+        read -p "[tue-get] Could not update tue-get. Continue? " -n 1 -r
+        exec <&-
+        echo    # (optional) move to a new line
+        if [[ ! $REPLY =~ ^[Yy]$ ]]
+        then
+            exit 1
+        fi
+    fi
+fi
+
+if [ ! -d $TUE_ENV_TARGETS_DIR ]
+then
+    git clone https://github.com/tue-robotics/tue-env-targets.git $TUE_ENV_TARGETS_DIR
```

Comment:
This is bound to cause problems.

Reply:
May be a good solution is to exit with error instead of cloning

Reply:
It is the same as tue-env. But throwing an error would be better

Reply:
Done